maggy/ablation/ablator/loco.py

```diff
@@ -213,12 +232,16 @@ def create_trial_dict(self, ablated_feature=None, layer_identifier=None):
             )
             trial_dict["ablated_feature"] = ablated_feature
 
-        if layer_identifier is None:
+        # 2 - determine the model generation logic
+        # 2.1 - no model ablation
+
+        if layer_identifier is None and custom_model_generator is None :
```

Comment:
```suggestion
        if not layer_identifier and not custom_model_generator:
```
your call, whatever you like more, but remove space before the colon